src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Azs.Backup.Admin.psd1

```diff
@@ -31,7 +31,7 @@ Changes may cause incorrect behavior and will be lost if the code is regenerated
     RootModule           = 'Azs.Backup.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion        = '0.2.0'
+    ModuleVersion        = '0.3.0'
```

Comment:
If we are going to have a breaking change, we should be updating the major version

Reply:
Version 1.0.0 sounds like not "preview" anymore. Is that fine?

Reply:
I think your change is fine.  We should eventually move towards a release version though.

Reply:
yes 0.3.0 is fine for now